guides/v2.0/extension-dev-guide/test-module.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+group: extension-dev-guide
+subgroup: 
```

Comment:
Two problems. `subgroup` needs to be there; otherwise, the topic displays right under the guide name on the left. Second, there are still two topics, one in `ext-dev-guide` and one in `validate`. Seems like there should be just one file and it should be under `validate`. If I'm offbase, please let me know.

Thanks for the outstanding contribution!!